Packs/Cryptocurrency/IndicatorFields/indicatorfield-Cryptocurrency_Address_Bitcoin_Abuse_Description.json

```diff
@@ -0,0 +1,30 @@
+{
+    "id": "indicator_bitcoinabusedescription",
+    "version": -1,
+    "modified": "2020-12-16T11:51:04.125836Z",
+    "name": "Bitcoin Abuse Description",
+    "ownerOnly": false,
+    "cliName": "bitcoinabusedescription",
+    "type": "shortText",
```

Comment:
shouldnt this be long text? how long can the description be?

Reply:
Where can i find whats shortText max chars and long text max chars?